frontend/components/DocumentFilters/DocumentFilters.jsx

```diff
@@ -140,9 +166,8 @@ export default class DocumentFilters extends Component {
     const {updateUrlParams} = this.props
 
     let validFilters = filters.filter(entry => {
-      let isValid = Object.keys(entry).filter(key => {
-        return Object.is(entry[key], null)
-      })
+      let isValid = Object.keys(entry)
+        .filter(key => Object.is(entry[key], null))
```

Comment:
Do we need `Object.is()` here? It's not supported by any version of IE and it seems that, in this case, we can just check if the value is falsy or compare against `null`.